### DISCUSSION TITLE
404 account not found

### DISCUSSION THREAD

**Discussion by arpy8** [community/community#66817] [Other Feature Feedback, Questions, & Ideas]
I am able to login into my account. Everything works fine until I'm logged in. 

But when I search for my account from someone else's device, the url shows "404 not found".

**Comment by Piero24** [comment]
@arpy8 you should check your username and email address settings.
This usually occurs when your username and email address are not configured correctly.
It may also be possible that you used an untrustworthy email address when registering.

Go to your account settings and see if you have the ability to change your username.
If you are unable to do this yourself and are prompted to contact support,
it means your account is restricted. In this case, contact GitHub support,
They will tell you how to proceed.
